app-frontend/src/app/pages/admin/team/users/users.js

```diff
@@ -79,6 +79,9 @@ class TeamUsersController {
             this.updatePagination(response);
             this.lastUserResult = response;
             this.users = response.results;
+            if (this.isEffectiveAdmin) {
```

Comment:
Not to be repetitive, but I think this logic would be better if incorporated into the `buildOptions` method.